SU-SOE/stanford_bean_types
# SOEOP-134: Fixing the top for chrome browsers.

# READY FOR REVIEW

# Summary
- Fix top spacing in Chrome to 65%

# Needed By (Date)
- Sprint end

# Criticality
- Client is eagerly awaiting this for the release.

# Steps to Test
- checkout branch `SOEOP-134`
- `drush cc css-js`
- go to the homepage to see the style changes
- Make sure things look good in chrome featured block. Refer to the screenshots in the JIRA ticket.

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOEOP-134

## Related PRs

## More Information

## Folks to notify

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
